libs/geo/src/main/java/org/elasticsearch/geometry/utils/SpatialUtils.java

```diff
@@ -4,19 +4,21 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-package org.elasticsearch.xpack.spatial;
+package org.elasticsearch.geometry.utils;
 
 import org.apache.lucene.util.SloppyMath;
 import org.elasticsearch.geometry.Circle;
 import org.elasticsearch.geometry.LinearRing;
 import org.elasticsearch.geometry.Polygon;
-import org.elasticsearch.index.mapper.GeoShapeIndexer;
 
 /**
  * Utility class for storing different helpful re-usable spatial functions
  */
 public class SpatialUtils {
```

Comment:
Can we change the name of the class now that it is in this library. Maybe CircleUtils?